pkg/kubelet/eviction/helpers.go

```diff
@@ -367,6 +367,16 @@ func parseThresholdStatements(statements map[string]string) ([]evictionapi.Thres
 			results = append(results, *result)
 		}
 	}
+	if utilfeature.DefaultFeatureGate.Enabled(features.InheritDefaultEvictionValues) {
```

Comment:
Please add a unit test for this function.

Reply:
yes @kannon92 , I am working on that